ProjectBank/Infrastructure.Tests/ProjectRepositoryTests.cs

```diff
@@ -21,119 +20,187 @@ public async Task CreateProject_given_CreateProjectDTO_returns_ProjectDTO()
         Assert.Equal("Algorithm", created.Title);
         Assert.Equal(Status.Active, created.Status);
         Assert.Equal(1, created.UserId);
+        Assert.Equal("Sorting", created.Description);
+        Assert.Equal(DateTime.UtcNow, created.CreationDate, precision: TimeSpan.FromSeconds(5));
+        Assert.Equal(DateTime.UtcNow, created.UpdatedDate, precision: TimeSpan.FromSeconds(5));
+        Assert.Equal(new List<string>() { "DMAT", "Fun" }, created.Tags);
+        Assert.Equal(new List<UserDTO>(), created.Participants);
     }
 
     [Fact]
     public async Task DeleteProjectById_given_ProjectId_deletes_projects()
     {
-        await _projectRepository.CreateProjectAsync(new CreateProjectDTO { Title = "Algo", UserId = 1 });
+        await _projectRepository.CreateProjectAsync(new CreateProjectDTO { Title = "Algo", UserId = 1, Description = "Very fun", Tags = new List<string> { "Sorting" } });
 
         await _projectRepository.DeleteProjectByIdAsync(1);
 
         Assert.Null(_context.Projects.Find(1));
     }
 
     [Fact]
-
     public async Task EditProject_given_CreateProjectDTO_updates_project()
```

Comment:
We might need two extra testcases for editing a project:
* One where we keep the existing tags and add more
* One where we remove all existing tags and add some new ones